src/client/automation/playback/click/select-child.js

```diff
@@ -42,8 +42,8 @@ export default class SelectChildClickAutomation {
             var isOption      = domUtils.isOptionElement(this.element);
             var selectedIndex = this.parentSelect.selectedIndex;
 
-            this.childIndex = isOption ? domUtils.getOptionIndex(this.parentSelect, this.element) :
-                              domUtils.getOptionGroupIndex(this.parentSelect, this.element);
+            this.childIndex = isOption ? domUtils.getElementIndexInParent(this.parentSelect, this.element) :
+                              domUtils.getElementIndexInParent(this.parentSelect, this.element);
```

Comment:
As I see the `getOptionGroupIndex` is used just in one more place. Let's kill it there too